syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode
{
    int32 errorcode = 1;
    bytes errormsg = 2;
}


message GetFriendListsRequest
{
    int32 id = 1;    
}

message GetFriendListsResponse
{
    ResultCode result = 1;
    repeated bytes name = 2;
}


service GetFriendListsServiceRpc
{
    rpc GetFriendLists(GetFriendListsRequest) returns(GetFriendListsResponse);
}